### PR TITLE
Update to v2 module

### PR DESCRIPTION
**Pull request by dmikusa**
## Summary

Update the release-2.x branch to use the `/v2` module.

## Use Cases

This is a precursor to cutting an alpha.1 release.

## Checklist

* [ ] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [ ] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [ ] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: buildpack_plan.go [hunk 1]
```diff
@@ -20,8 +20,8 @@ import (
 	"fmt"
 	"reflect"
 
+	"dario.cat/mergo"
```
Comment (anthonydahanne) [inline buildpack_plan.go:23]: why the change?

**Review by anthonydahanne — APPROVED** [review]

File: buildpack_plan.go (hunk 1 above)
Comment (dmikusa) [reply buildpack_plan.go:23]: It came up as needing an update, so I did. Nothing beyond that. I don't think we need to, but we're in a breaking change window, so it seemed like a good time.
